streams/src/main/java/org/apache/kafka/streams/processor/internals/StandbyTask.java

```diff
@@ -108,13 +107,20 @@ public void completeRestoration() {
     }
 
     @Override
-    public void prepareSuspend() {
-        log.trace("No-op prepareSuspend with state {}", state());
+    public void suspendDirty() {
+        log.trace("No-op suspend dirty with state {}", state());
+        if (state() == State.RUNNING) {
```

Comment:
Do we want to throw here if the current state is `CLOSED`?

Reply:
My proposal is, to keep the methods idempotent.

Reply:
Cool 👍